iceoryx_utils/source/posix_wrapper/timer.cpp

```diff
@@ -610,4 +610,39 @@ cxx::error<TimerError> Timer::createErrorFromErrno(const int32_t errnum) noexcep
 }
 
 } // namespace posix
+
+namespace clock
+{
+Timer::Timer(const iox::units::Duration timeToWait) noexcept
+    : m_measuringDuration(timeToWait)
```

Comment:
I think this two variables should have a similar name, since they are the same. I like `timeToWait` more ... it's shorter

Reply:
Okay m_measuringDuration is changed to timeToWait